packages/libraries/cli/package.json

```diff
@@ -18,32 +19,49 @@
     "node": ">=14.0.0"
   },
   "bin": {
-    "hive": "./bin/run"
+    "hive": "dist/cjs/bin.js"
   },
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "require": {
+        "types": "./dist/typings/index.d.cts",
+        "default": "./dist/cjs/index.js"
+      },
+      "import": {
+        "types": "./dist/typings/index.d.ts",
+        "default": "./dist/esm/index.js"
+      },
+      "default": {
+        "types": "./dist/typings/index.d.ts",
+        "default": "./dist/esm/index.js"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "typings": "dist/typings/index.d.ts",
   "files": [
-    "/bin",
-    "/dist",
-    "/oclif.manifest.json"
+    "/dist"
   ],
   "keywords": [
     "graphql"
   ],
   "scripts": {
-    "build": "tsc",
-    "oclif:pack": "npm pack && pnpm oclif pack tarballs --no-xz && pnpm oclif pack win",
-    "oclif:upload": "pnpm oclif upload tarballs --no-xz && pnpm oclif upload win",
-    "postpack": "rm -f oclif.manifest.json",
+    "build": "pnpm tsx ./scripts/version.ts && bob build --single && tsx ../../../scripts/runify.ts",
+    "compile": "pnpm tsx ./scripts/node-to-deno.ts && pnpm compile:lin && pnpm compile:mac && pnpm compile:win",
+    "compile:lin": "deno compile -A deno/index.ts --no-check --output binaries/hive-linux   --target x86_64-unknown-linux-gnu",
+    "compile:mac": "deno compile -A deno/index.ts --no-check --output binaries/hive-macos   --target x86_64-apple-darwin",
+    "compile:win": "deno compile -A deno/index.ts --no-check --output binaries/hive-windows --target x86_64-pc-windows-msvc",
```

Comment:
what about arm?

Reply:
only available arm target is for macos

Reply:
hmm, we already use arm linux everywhere possible, we need to make sure others that do the same can still use the cli